source/core/tests/unit_tests/gold/application/streams/test_calclated_measurements_stream.py

```diff
@@ -0,0 +1,49 @@
+from unittest.mock import Mock, patch
+
+from pyspark.sql import SparkSession
+
+import core.gold.application.streams.calculated_measurements_stream as sut
+from core.gold.infrastructure.repositories.calculated_measurements_repository import CalculatedMeasurementsRepository
+from core.gold.infrastructure.repositories.gold_measurements_repository import GoldMeasurementsRepository
+from core.gold.infrastructure.streams.gold_measurements_stream import GoldMeasurementsStream
+
+
+def test__stream_measurements_calculated_to_gold__calls_expected(spark: SparkSession):
+    # Arrange
+    calculated_repo_mock = Mock(spec=CalculatedMeasurementsRepository)
+    gold_stream_mock = Mock(spec=GoldMeasurementsStream)
+    with (
+        patch.object(sut, "CalculatedMeasurementsRepository", return_value=calculated_repo_mock),
+        patch.object(sut, "GoldMeasurementsStream", return_value=gold_stream_mock),
+    ):
```

Comment:
```suggestion
def test__stream_measurements_calculated_to_gold__calls_expected(spark: SparkSession, mocker: MockerFixture):
    # Arrange
    calculated_repo_mock = Mock(spec=CalculatedMeasurementsRepository)
    gold_stream_mock = Mock(spec=GoldMeasurementsStream)
    mocker.patch.object(sut, "CalculatedMeasurementsRepository", return_value=calculated_repo_mock)
    mocker.patch.object(sut, "GoldMeasurementsStream", return_value=gold_stream_mock)
```

use the MockerFixture instead. Goes for all tests with mocking